k8s/README.md

```diff
@@ -2,56 +2,32 @@
 
 ![Nameko loves Kubernetes](nameko-k8s.png)
 
-In this example we'll use local [minikube](https://github.com/kubernetes/minikube)
-Kubernetes cluster hosted on VirtualBox along with community maintained Helm Charts to deploy all 3rd party services. We will also create a set of Helm Charts for Nameko Example Services from this repository.  
+In this example we'll use local Kubernetes cluster installed with [docker-for-desktop](https://docs.docker.com/docker-for-mac/)
+ along with community maintained Helm Charts to deploy all 3rd party services. 
+ We will also create a set of Helm Charts for Nameko Example Services from this repository.  
 
-Tested with Kubernetes v1.8.
+Tested with Kubernetes v1.14.7
 
 ## Prerequisites
 
 Please make sure these are installed and working
 
-* [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* [Minikube](https://github.com/kubernetes/minikube#installation)
+* [docker-for-desktop](https://docs.docker.com/docker-for-mac/) with Kubernetes enabled
```

Comment:
Is there a non Mac-specific version of this page?

Reply:
Not really, but I can point to both [mac](https://docs.docker.com/docker-for-mac/) and [windows](https://docs.docker.com/docker-for-windows/)? Although no way I can test this on Windows at the moment.

Reply:
Perhaps just add a comment saying we assume Mac for simplicity but it should work on Windows or indeed any other Kubernetes cluster.